c_emulator/riscv_platform_impl.c

```diff
@@ -6,6 +6,9 @@
 uint64_t rv_pmp_count = 0;
 uint64_t rv_pmp_grain = 0;
 
+uint64_t rv_vector_vlen_exp = 0x7;
```

Comment:
Sorry I think I told you the wrong thing before; to match the existing value this should be 9.